packages/wrangler/src/dev/local.tsx

```diff
@@ -255,7 +255,7 @@ function useLocalWorker({
 			});
 			//TODO: instead of being lucky with spawn's timing, have miniflare-cli notify wrangler that it's ready in packages/wrangler/src/miniflare-cli/index.ts, after the mf.startScheduler promise resolves
 			if (onReady) {
-				await new Promise((resolve) => setTimeout(resolve, 200));
+				await new Promise((resolve) => setTimeout(resolve, 5000));
```

Comment:
is this where the 200 ms delay came from? also now it's 5 seconds?

Reply:
Yes - since we haven't implemented message passing between the wrangler and miniflare processes, this is a hack which waits 5000ms for miniflare to be ready for requests. Will try lower values and see if they pass.

Reply:
Can be as low as 250ms, but it's flaky. 500ms seems to be a good spot for now.